core/src/main/java/io/grpc/ChannelImpl.java

```diff
@@ -198,9 +205,11 @@ public ChannelImpl shutdown() {
    *
    * <p>NOT YET IMPLEMENTED. This method currently behaves identically to shutdown().
    */
-  // TODO(ejona86): cancel preexisting calls.
   public ChannelImpl shutdownNow() {
     shutdown();
+    for (ClientCall<?, ?> call : activeCalls) {
+      call.cancel();
```

Comment:
It needs to cancel the streams, not the calls. Cancelling the call causes state changes to enforce:
> No other methods on this class can be called after this method has been called.